spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/AbstractEntity.java

```diff
@@ -38,6 +39,10 @@
 	@JsonIgnore
 	private final Long id;
 
+	@Version
+	@JsonIgnore
+	private Long objectVersion;
```

Comment:
nitpicky... I know... Why objectVersion instead of just version?

Reply:
Nevermind.  Just read comment on it in Slack.

Reply:
`PackageMetadata` has a `version` (String) property.